packages/core/src/App/Components/Layout/Header/__tests__/platform-switcher.spec.tsx

```diff
@@ -52,4 +62,10 @@ describe('PlatformSwitcher component', () => {
         const div_element = screen.getByTestId('dt_platform_switcher');
         expect(div_element).toHaveClass('platform-switcher--is-mobile');
     });
+
+    it('should have "platform-switcher--dark-mode" class if "app_routing_history" is not an empty array and "is_dark_mode" is "false"', () => {
```

Comment:
This sounds weird. Shouldn't `"platform-switcher--dark-mode"` be there if `"is_dark_mode" is "true"'` ?

Reply:
Since we need to remove the dark mode class. we will remove this test case.